zone/npc.cpp

```diff
@@ -1246,11 +1290,11 @@ uint32 ZoneDatabase::CreateNewNPCCommand(
 
 	s2.zone         = zone;
 	s2.version      = instance_version;
-	s2.x            = n->GetX();
-	s2.y            = n->GetY();
-	s2.z            = n->GetZ();
-	s2.respawntime  = 1200;
-	s2.heading      = n->GetHeading();
+	s2.x            = c->GetX();
+	s2.y            = c->GetY();
+	s2.z            = c->GetZ();
+	s2.respawntime  = extra > 0 ? extra : 1200;
+	s2.heading      = c->GetHeading();
```

Comment:
This changes functionality.

Currently if you spawn an NPC and use `#npcspawn create` it uses the NPC's position for where to spawn it, and most operators are moving around as they're doing this.

Changing this to use the client's position will change the way operators have to think about placing NPCs as their position will now be relative to their own rather than where they spawned the NPC.

Would prefer if we leave this alone and continue to use the NPC's position in case you spawn 5 NPCs then need to create spawns for them all they're not all using your current location.

Reply:
Understandable, reverted in new [commit](https://github.com/EQEmu/Server/pull/4311/commits/6524e60ba154786a0f7a5727e0717644fbd6a1d3) - I left clone as using the clients position though as that makes the most sense to me (walking around hitting #npccreate clone to populate an area as opposed to having to #repop and #spawnfix each time). I noted this in the updated help text as well